rna/rna-index/README.md

```diff
@@ -0,0 +1,29 @@
+# RNAIndex with Samtools
```

Comment:
Let's rename this directory to `rna-bam-sort-and-index` and workflow `RNABAMSortAndIndex`